ipu/examples/jax_ipu_mnist_neural_net_training.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# JAX on IPU: MNIST neural net training\n",
+    "\n",
+    "MNIST neural net training on IPU, following the original JAX tutorial: https://jax.readthedocs.io/en/latest/notebooks/Neural_Network_and_Data_Loading.html\n",
+    "\n",
+    "The original example will run as-is on the IPU. Nevertheless, to make full advantage of the IPU hardware and improve user experience, the following modifications were made:\n",
+    "\n",
+    "* `jax_platforms = \"cpu,ipu\"`: use CPU as the default platform for initialization of parameters and dataset.\n",
```

Comment:
Add "TODO: remove when PRNG lands"?

Reply:
Even when the custom IPU PRNG lands, I believe initialization of parameters will always be more user friendly on the host (due to compilation time + engine run latency). There is just not enough to do for the accelerator to really see the benefit of using it.